webview/src/setup/components/dvc/Dvc.tsx

```diff
@@ -36,9 +38,21 @@ export const Dvc: React.FC<DvcProps> = ({
   needsGitInitialized,
   projectInitialized,
   pythonBinPath,
-  setSectionCollapsed,
-  isExperimentsAvailable
+  hasReceivedMessageFromVsCode,
+  setSectionCollapsed
 }) => {
+  const isSetup = projectInitialized && !!cliCompatible
+
+  const onCompletion = () => {
+    setSectionCollapsed({
```

Comment:
I decided just to leave experiments as is instead of redirecting the page, since in my opinion, it feels more natural for a redirection if there was nothing else to do on the Setup page which isn't the case since there's a studio section and more sections coming. 

But no strong preference, opening experiments does get the user a little quicker to DVC features.  With the reusable hook, it's simple enough to do so if we'd like!